plugins/nanospell/plugin.js

```diff
@@ -429,23 +416,36 @@
 				triggerSpelling((spellFastAfterSpacebar && (ch8r === CHARCODES.SPACE || ch8r === CHARCODES.LF || ch8r === CHARCODES.CR)), target)
 			}
 
+			function findNearestParentBlock(element) {
+				var elementPath = new CKEDITOR.dom.elementPath(element),
+					elements = elementPath.elements,
+					i;
+
+				for (i in elements) {
```

Comment:
Is this actually consistent?  

Memory is foggy, but elements is an array, right?  

See https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/for...in#Array_iteration_and_for...in